aviary/utils.py

```diff
@@ -202,45 +198,45 @@ def init_losses(
 
     Args:
         task_dict (dict[str, TaskType]): Map of target names to "regression" or "classification".
-        loss_dict (dict[str, "L1" | "L2" | "CSE"]): Map of target names to loss functions.
+        loss_dict (dict[str, "L1" | "L2" | "CSE"]): Map of target names to loss type.
         robust (bool, optional): Whether to use an uncertainty adjusted loss. Defaults to False.
 
     Returns:
         dict[str, tuple[str, type[torch.nn.Module]]]: Dictionary of losses for each task
     """
-    criterion_dict: dict[str, tuple[str, type[torch.nn.Module]]] = {}
+    _loss_dict: dict[str, tuple[str, type[torch.nn.Module]]] = {}
```

Comment:
I don't necessaryily think `_loss_dict` is better than `criterion_dict` maybe `loss_fn_dict` and `loss_name_dict` or something to make clear one is a dict of strings and the other a dict of objects/functions

Reply:
Good idea, I like those names.